### ISSUE TITLE
Multiple HCPCS standard concepts for "Patients who died from cancer"

### ISSUE THREAD

**Issue by Spayralbe** [OHDSI/OMOP-Standardized-Vocabularies#42]
Among the child concepts of "Death" (4306655), there are 5 HCPCS standard concepts that only differ in concept code/id:

Patients who died from cancer | 1314519 | HCPCS
Patients who died from cancer | 1314522 | HCPCS
Patients who died from cancer | 1314525 | HCPCS
Patients who died from cancer | 1314528 | HCPCS
Patients who died from cancer | 1314532 | HCPCS

Wouldn't it be better to make one of these the OMOP standard concept and let the other four map to that one?

**Comment by dimshitc** [comment]
Yeah. That's a known HCPCS issue. There are two things - real duplicates
and those that look like duplicates but in reality they have different
concept names. We need to get all the full names and then deal with
duplicates the way you suggested (if there still be any after names
enhancement).
We have this in our backlog. We'll let you know when there's a progress.
